nix/pkgs/default.nix

```diff
@@ -56,15 +58,16 @@ let
   '';
 
   ci = callPackage ./ci.nix { };
-  core = callPackage ./core.nix {
-    inherit hcToplevelDir;
-    releaseAutomation = "${hcReleaseAutomation}/bin/hc-ra";
-  } // {
+  core = callPackage ./core.nix
+    {
+      inherit hcToplevelDir;
+      releaseAutomation = "${hcReleaseAutomation}/bin/hc-ra";
+    } // {
     inherit hcReleaseAutomation;
   };
   happ = {
-    holochain = mkHolochainBinaryScript "holochain";
-    hc = mkHolochainBinaryScript "hc";
+    holochain = mkHolochainBinaryScript "holochain" "";
+    hc = mkHolochainBinaryScript "hc" "--release";
```

Comment:
Why does `hc` get a release build but not `holochain`?

Reply:
i'd prefer if we wouldn't build `hc` with `--release` either in this case. the point is that these wrapper binaries will be used in debugging contexts locally, most likely by holochain core devs who iterate on the core code while checking how the code changes integrate into other apps they try out locally.

the point of this PR is to make `hc` work at all in this context. the issue is that its debug build currently produces broken runtime behavior. that is an issue to take care off later, this one makes it work in the meantime.

Reply:
I'm curious what the problem is. Can you put a comment explaining the problem here so we know that this is a temporary fix, and what will enable us to remove it?

Reply:
the commit message has some details, except the actual runtime error that occurs on debug builds. here it is:

```
+./target/debug/hc --version
thread 'main' panicked at 'Argument short must be unique

    -o is already in use', ./.cargo/registry/src/github.com-1ecc6299db9ec823/clap-2.34.0/src/app/parser.rs:190:13
```

Reply:
@maackle ping

Reply:
I feel like we should just fix this in `hc`, looks like an easy fix

Reply:
FYI I am able to run a debug build of `hc` just fine locally on develop, i.e. this works with no error:

```
cargo build --bin hc && ./target/debug/hc --version
```

Reply:
indeed! i tried it out just now and it has been (coincidentally) repaired in the meantime